script/authors.go

```diff
@@ -117,7 +117,7 @@ func main() {
 
 	authorsRe := regexp.MustCompile(`(?s)id="contributor-list">.*?</div>`)
 	bs := readAll(htmlFile)
-	bs = authorsRe.ReplaceAll(bs, []byte("id=\"contributor-list\">\n"+replacement+"\n    </div>"))
+	bs = authorsRe.ReplaceAll(bs, []byte("id=\"contributor-list\">\n"+replacement+"\n          </div>"))
```

Comment:
Just checking: Is this change on purpose?

Reply:
Yes it is, matching the container restructuring in that dialog.